bigquery_etl/parse_metadata.py

```diff
@@ -29,7 +29,13 @@ def is_valid_label(label):
         63 characters, and cannot be empty. Values can be empty, and have
         a maximum length of 63 characters.
         """
-        return re.match(r"[\w\d_-]+", label) and len(label) <= 63
+        return (
+            len(label) > 0
+            and re.match(r"[\w\d_-]+", label)
+            and len(label) <= 63
+            and label.lower() == label
```

Comment:
It might be better to build this into the regex. `re.fullmatch(r"[0-9a-z-_]{1,63}")` would handle all of these checks, I believe. It would disallow international characters, but it feels safer to disallow those anyway (and note as much in the docstring).

Reply:
Yeah, the simplification is probably worth not allowing international characters